tests/embedding/test_app.py

```diff
@@ -44,9 +44,10 @@ def test_app_repr(embedded_app):
 @pytest.mark.embedding
 @pytest.mark.minimum_version(241)
 def test_deprecation_warning(embedded_app):
-    struct = embedded_app.Model.AddStaticStructuralAnalysis()
+    harmonic_acoustic = embedded_app.Model.AddHarmonicAcousticAnalysis()
     with pytest.warns(UserWarning):
-        struct.SystemID
+        harmonic_acoustic.SystemID
+        harmonic_acoustic.AnalysisSettings.MultipleRPMs = True
```

Comment:
minor - putting both in a `pytest.warns` will technically allow the test to pass if one of these does not cause a deprecation warning. A better design would be to put each in its own `pytest.warns` block

Reply:
I already merged this. But I will make these changes in the other PR